platforms/ios/AppPlatform_iOS.h

```diff
@@ -0,0 +1,58 @@
+//
+//  AppPlatform_iOS.h
+//  Minecraft
+//
+//  Created by Brent on 10/30/23.
+//  Copyright (c) 2023 __MyCompanyName__. All rights reserved.
+//
+
+#pragma once
+
+#import "minecraftpeViewController.h"
+
+#include <string>
+
+#include "client/app/AppPlatform.hpp"
+
+#include "client/player/input/Mouse.hpp"
+#include "client/player/input/Keyboard.hpp"
+#include "common/Logger.hpp"
+
+class AppPlatform_iOS : public AppPlatform
+{
+public:
+    AppPlatform_iOS(minecraftpeViewController *viewController);
+    ~AppPlatform_iOS();
+    
+	void initSoundSystem() override;
+    
+	int checkLicense() override;
+	int getScreenWidth() const override;
+	int getScreenHeight() const override;
+	Texture loadTexture(const std::string& path, bool b = false) override;
+	int getUserInputStatus() override;
+    bool isTouchscreen() override;
+    std::string getAssetPath(const std::string &path) const override;
+    std::string getPatchData() override;
```

Comment:
Inconsistend indentation

Reply:
This looks totally fine in any normal text editor. For some reason web browsers decide that tabs are not 4 spaces, but are actually significantly longer. @iProgramMC should we be using tabs or spaces? Xcode has been automatically using spaces.